src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotRight = {};
+
+  for (const key in robot) {
+    if (robot[key] in robotRight) {
```

Comment:
Good! For some reason hasOwnProperty is better. See checklist p2

Reply:
Yes, I saw it in the checklist, but in principle I will leave this option, it fits a good example